demos/storybook/stories/dropdown-toolbar/with-basic-usage.tsx

```diff
@@ -0,0 +1,27 @@
+import { DropdownToolbar } from '@pxblue/react-components';
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
+import React from 'react';
+import { WITH_MIN_PROPS_STORY_NAME } from '../../src/constants';
+import { text } from '@storybook/addon-knobs';
+import { AppBar } from '@material-ui/core';
+import { HandlerFunction, action } from '@storybook/addon-actions';
+
+export const withBasicUsage = (): StoryFnReactReturnType => {
+    const menuItems = [
+        { label: 'English', onClick: (): HandlerFunction => action('English selected') },
+        { label: 'Arabic', onClick: (): HandlerFunction => action('Arabic selected') },
+        { label: 'French', onClick: (): HandlerFunction => action('French selected') },
```

Comment:
Can we make these more generic since we aren't hooking them up to state ("Item 1, Item 2, Item3")?

The actions do not appear to be showing up in the actions tab